osmosis-1/osmosis-1.assetlist.json

```diff
@@ -5083,6 +5083,42 @@
         "OSMO": 857
       }
     },
+    {
+      "description": "The native EVM, governance and staking token of the Imversed",
+      "denom_units": [
+        {
+          "denom": "ibc/92B223EBFA74DB99BEA92B23DEAA6050734FEEAABB84689CB8E1AE8F9C9F9AF4",
+          "exponent": 0,
+          "aliases": [
+            "aimv"
+          ]
+        },
+        {
+          "denom": "imv",
+          "exponent": 18
+        }
+      ],
+      "base": "ibc/92B223EBFA74DB99BEA92B23DEAA6050734FEEAABB84689CB8E1AE8F9C9F9AF4",
+      "name": "Imversed",
+      "display": "imversed",
+      "symbol": "IMV",
+      "ibc": {
+        "source_channel": "channel-1",
+        "dst_channel": "channel-517",
+        "source_denom": "aimv"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/imversed/images/imversed.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/imversed/images/imversed.svg"
+      },
+      "coingecko_id": "imv",
```

Comment:
This "id" doesn't seem to exist when I checked CoinGecko--does this exist?

Reply:
No, we don't register this id on CoinGecko yet. Should I remove it from json?